packages/workflows/src/cli/list/activeWorkflows/ActiveWorkflows.handler.ts

```diff
@@ -0,0 +1,66 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { IHandlerParameters } from "@brightside/imperative";
+import { ListWorkflows } from "../../../api/ListWorkflows";
+import { ZosmfBaseHandler } from "../../../../../zosmf/src/ZosmfBaseHandler";
+import { IActiveWorkflows } from "../../../api/doc/IActiveWorkflows";
+
+
+/**
+ * Common Handler for creating workflow instance in z/OSMF in zosworkflows package.
+ * This is not something that is intended to be used outside of this npm package.
+ */
+
+export default class ListActiveWorkflowHandler extends ZosmfBaseHandler {
```

Comment:
should this be ListActiveWorkflowHandler  or ListActiveWorkflowsHandler, since it can list multiple workflows ?